src/test/java/com/github/dockerjava/api/model/InfoTest.java

```diff
@@ -169,4 +170,161 @@ public void serder1Json() throws IOException {
 
         assertThat(info, is(withInfo));
     }
+
+    @Test
+    public void serder2Json() throws IOException {
+        final ObjectMapper mapper = new ObjectMapper();
+        final JavaType type = mapper.getTypeFactory().constructType(Info.class);
+
+        final Info info = testRoundTrip(VERSION_1_22,
+                "info/2.json",
```

Comment:
Could you rename filename to something meaningful to initial issue that you had? 
I called initial dump as 1.json because it was the first real dump, but it seems was bad idea to name file in such way.